docs/machine-learning/tutorials/sentiment-analysis-cli.md

```diff
@@ -152,62 +152,56 @@ Those enumerated assets are explained in the following steps of the tutorial.
-
-    - In the fourth line of code you load create the `PredictionEngine` object with the `mlContext.Model.CreatePredictionEngine<TSrc,TDst>(ITransformer mlModel)` API. You need the `PredictionEngine` object whenever you want to make a prediction targeting a single sample of data (In this case, a single piece of text to predict its sentiment).
-
-    - The fifth line of code is where you create that *single sample data* to be used for the prediction by calling the function `CreateSingleDataSample()`. Since the CLI tool doesn't know what kind of sample data to use, within that function it is loading the first row of the dataset. However, for this case you can also create you own 'hard-coded' data instead of the current implementation of the `CreateSingleDataSample()` function by updating this simpler code implementing that function:
+    - The first lines of code simply create a *single sample data*, in this case based on the first row of your dataset to be used for the prediction. You can also create you own 'hard-coded' data by updating the code:
 
         ```csharp
-        private static ModelInput CreateSingleDataSample()
+        ModelInput sampleData = new ModelInput()
         {
-            ModelInput sampleForPrediction = new ModelInput() { Col0 = "The ML.NET CLI is great for getting started. Very cool!", Label = true };
-            return sampleForPrediction;
-        }
-        ```
+            Col0 = "The ML.NET CLI is great for getting started. Very cool!"
+        };
+        ```    
+
+    - The next line of code uses the `ConsumeModel.Predict()` method on the specified input data to make a prediction and return the results (based on the ModelOutput.cs schema).
+    
+    - The last lines of code print out the proprties of the sample data (in this case the Comment) as well as the Sentiment prediction and corresponding Scores for positive sentiment (1) and negative sentiment (2).
 
 1. Run the project, either using the original sample data loaded from the first row of the dataset or by providing your own custom hard-coded sample data. You should get a prediction comparable to:
 
     # [Windows](#tab/windows)
 
     Run the console app from Visual Studio by hitting F5 (Play button):
 
-    ![ML.NET CLI auto-train on PowerShell](./media/mlnet-cli/sample-cli-prediction-execution.png))
+    ![ML.NET CLI run the app from Visual Studio](./media/mlnet-cli/mlnet-cli-console-app.png))
 
     # [macOS Bash](#tab/macosbash)
 
     Run the console app from the command-prompt by typing the following commands:
 
     ```dotnetcli
-    cd SampleBinaryClassification
-    cd SampleBinaryClassification.ConsoleApp
+    cd SampleClassification
+    cd SampleClassification.ConsoleApp
 
     dotnet run
     ```
 
-    ![ML.NET CLI auto-train on PowerShell](./media/mlnet-cli/sample-cli-prediction-execution-bash.png))
+    ![ML.NET CLI run the app in bash](./media/mlnet-cli/sample-cli-prediction-execution-bash.png))
```

Comment:
The Mac image is not up do date, I would suggest updating or removing it

Reply:
Yep that's the one TO DO for now. We haven't been able to get the current update of CLI working on Mac